space.rirusha.Cassette.yml

```diff
@@ -0,0 +1,53 @@
+id: space.rirusha.Cassette
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.vala
+command: cassette
+finish-args:
+- --share=network
+- --share=ipc
+- --socket=fallback-x11
+- --device=dri
+- --socket=pulseaudio
+- --socket=wayland
+- --filesystem=xdg-music:ro
+- --filesystem=xdg-download:ro
+- --talk-name=org.mpris.MediaPlayer2.Player
+- --own-name=org.mpris.MediaPlayer2.cassette
```

Comment:
Can you make this use `org.mpris.MediaPlayer2.space.rirusha.Cassette` ? Then this permission can be dropped.

Reply:
Done